### DISCUSSION TITLE
Temporary validation control

### DISCUSSION THREAD

**Discussion by listepo** [CommunityToolkit/Maui#445] [New Feature Discussions]
I think it would be nice to have something like that https://github.com/CommunityToolkit/WindowsCommunityToolkit/issues/3862,

**Comment by bijington** [comment]
@listepo we provide a set of validators that can be attached to controls see here for example: https://docs.microsoft.com/en-gb/dotnet/communitytoolkit/maui/behaviors/text-validation-behavior

The approach is slightly different to your link but I think it should satisfy most use cases. Do you think see other use cases?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
